src/api/ratelimits.mdx

```diff
@@ -5,7 +5,8 @@ sidebar_order: 3
 
 Sentry rate limits every API request made to prevent abuse and resource overuse. The limit is applied to each unique combination of caller and endpoint.
 
-The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected.
+The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. The concurrent rate limit limits the number of requests happening at the same time to a given endpoint grouping. If the number of requests from a caller exceeds the number of allowed requests or concurrent requests within a window, then that request will be rejected.
+
```

Comment:
Ugh words are hard, especially when I barely have a grasp on the concept. Sorry if this sucks.